frontend/src/pages/DropGem/DropGem.vue

```diff
@@ -47,6 +60,28 @@ export default defineComponent({
         this.currentStage -= 1;
       }
     },
+    // Tried to make this inline, couldn't figure out how to pass in value.
+    setReceiverId(value: number): void {
+      this.draftGem.setReceiverId(value);
+    },
+    setMessage(value: string): void {
+      this.draftGem.setMessage(value);
+    },
+    setGemColor(value: GemColor): void {
+      this.draftGem.setColor(value);
+    },
+    dropMyGem() {
+      console.log("called dropMyGem");
+      const dropGem = () => this.store.dispatch("dropGem", this.draftGem);
+      const response = ref<number>(-1);
+      console.log(this.draftGem);
+      dropGem()
+        .then((resp) => {
+          response.value = resp;
+        })
+        .catch((err) => console.log(err, "Failed to drop gem"));
+      console.log(response.value);
```

Comment:
Should these `console.log` statements be removed?

Reply:
removed